### PR TITLE
Added an individual participant view

### PR DESCRIPTION
**Pull request by brockboland**
Dependent on https://github.com/vokal/ExtraLifeTeamInfo/pull/1

This isn't interesting on it's own: it just shows the progress for one person. I'm adding it so that I can add a corresponding view to the iOS app for demonstrating universal links.

@vokal/developers another quick-review one

### CODE REVIEW

**Comment by chillpop** [comment]
sure

File: views/person.ejs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		<script>
+			window.setTimeout(function(){location.reload();}, 900000); // Refresh every 15 minutes
+		</script>
+
+	</head>
+
+	<body>
+
+		<header class="member">
+			<a target="_blank" href="http://www.extra-life.org/index.cfm?fuseaction=donorDrive.participant&	participantID=<%= participantID %>" class="<%= percentageTowardGoal === 1 ? 'met' : '' %> <%= percentageTowardGoal > 1 ? 'exceed' : '' %>">
```
Comment (chillpop) [inline views/person.ejs]: should there be a space after `participant&`?

**Comment by chillpop** [comment]
just that one question about the URL

File: views/person.ejs (hunk 1 above)
Comment (brockboland) [inline views/person.ejs]: Whoops!

**Comment by brockboland** [comment]
Fixed. Cool?

**Comment by chillpop** [comment]
LGTM

**Comment by brockboland** [comment]
Thanks! Feel like reviewing https://github.com/vokal/ExtraLifeTeamInfo/pull/1? This one is dependent on that one.

**Pull request merged**